hack/record-gcp

```diff
@@ -27,7 +27,7 @@ echo "TEST_BILLING_ACCOUNT_ID=${TEST_BILLING_ACCOUNT_ID}"
 export TEST_BILLING_ACCOUNT_ID
 
 if [[ -z "${KCC_USE_DIRECT_RECONCILERS:-}" ]]; then
-  KCC_USE_DIRECT_RECONCILERS="ComputeForwardingRule"
+  KCC_USE_DIRECT_RECONCILERS="ComputeForwardingRule,SpannerInstance"
```

Comment:
Suggest not changing this env var.  We don't run real gcp against presubmit, so this is more like a local setup.